srcpkgs/drawing/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'drawing'
+pkgname=drawing
+version=1.0.1
+revision=1
+wrksrc=drawing-$version
+build_style=meson
+build_helper=gir
+hostmakedepends="python3 glib-devel appstream-glib pkg-config gettext itstool"
+makedepends="libglib-devel gtk+3-devel"
+depends="python3-gobject python3-cairo"
+short_desc="Simple image editor, similar to Microsoft Paint, for GNOME desktop."
+maintainer="mrkcee <markz_a@yahoo.com>"
+license="GPL-3.0-only"
```

Comment:
[from what I can see](https://github.com/maoschanz/drawing/search?q=any+later), this should be `GPL-3.0-or-later`